app/views/drawings/_drawings.html.haml

```diff
@@ -1 +1,51 @@
-= render @drawings.desc
+.container
+  %h3 Manage Drawings
+  %table.table.table-bordered.table-condensed.table-hover.table-responsive
+    %thead
+      %tr
+        %th ID
+        %th File
+        %th Subject Matter
```

Comment:
The Story is also "background story of the artist" so possibly that too. Not sure what language we can use to make that clearer?